repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/connectionframework/ConnectionFrameworkTest.groovy

```diff
@@ -62,7 +63,7 @@ class ConnectionFrameworkTest extends ReposeValveTest {
         messageChain = deproxy.makeRequest(url: reposeEndpoint + "/", headers: headers, addDefaultHeaders: false)
 
         then:
-        messageChain.handlings[0].request.headers.getFirstValue("accept") == null
+        StringUtils.isBlank(messageChain.handlings[0].request.headers.getFirstValue("accept"))
```

Comment:
Nothin' wrong here, just calling attention to it since it demonstrates the Jetty behavior change.

Reply:
Yep. Jetty v9.2.x supports HTTP/1.1 ([RFC 2616](https://tools.ietf.org/html/rfc2616)), javax.websocket, and SPDY v3. Jetty v9.3.x supports HTTP/1.1 ([RFC 7230](https://tools.ietf.org/html/rfc7230)), HTTP/2 (RFC 7540), WebSocket (RFC 6455, JSR 356), and FastCGI. One of the differences between HTTP/1.1 as defined by RFC 2616 and RFC 7230 is the way headers are defined and handled.

Specifically empty header values are now returned instead of a null. That is there is no longer a differentiation between an empty header and a header with an empty string as the value.